cloud/amazon/ec2_elb_lb.py

```diff
@@ -708,6 +719,12 @@ def _set_connection_draining_timeout(self):
             attributes.connection_draining.enabled = False
             self.elb_conn.modify_lb_attribute(self.name, 'ConnectionDraining', attributes.connection_draining)
 
+    def _set_idle_timeout(self):
+        attributes = self.elb.get_attributes()
+        if self.idle_timeout is not None and self.idle_timeout != attributes.connecting_settings.idle_timeout:
+            attributes.connecting_settings.idle_timeout = self.idle_timeout
+            self.elb_conn.modify_lb_attribute(self.name, 'connectingsettings',attributes.connecting_settings)
```

Comment:
Minor formatting issue: Add space between `'connectingSettings',` and `attributes`